website/docs/components/button/partials/code/how-to-use.md

```diff
@@ -77,6 +77,27 @@ This indicates that the Button should take up the full-width of the parent conta
 <Hds::Button @text="Full width button" @isFullWidth={{true}} />
 ```
 
+### Layout
+
+To change the default `block` layout to `inline`, set `@isLine` to `true`.
+
+In contexts where the Button needs to be <em>inline</em>, to inherit the alignment from a parent, you can use the `@isInline` argument
```

Comment:
I see the two paragraph as a bit of duplication. What do you think of merging them in a single concept?

```suggestion
To change the default `block` layout to `inline`, set `@isLine` to `true`. This can be useful in contexts where the Button needs to be <em>inline</em>, for example to inherit the alignment from a parent.
```